src/util.mts

```diff
@@ -1,6 +1,7 @@
 import { execSync } from "child_process";
-import { chownSync, existsSync, mkdirSync, statSync, writeFileSync } from "fs";
+import { existsSync, mkdirSync, statSync, writeFileSync } from "fs";
 import { join } from "path";
+import { AnsiEscapes, PRIV_CMD_EXEC } from "../scripts/inject/util.mjs";
```

Comment:
Can you move these things into this file and import them in the injector?

Reply:
You want AnsiEscapes and PRIV_CMD_EXEC inside of src/util.mts? They're only used in 1 location in the src/util.ts but are used many times in the scripts/inject/util.mjs but if you'd like I can. Or alternatively, did you want it in both scripts/inject/util.mjs and src/util.mts?

Reply:
> You want AnsiEscapes and PRIV_CMD_EXEC inside of src/util.mts?
Yes. From a structure POV, I think it's better to not be importing from scripts.